### ISSUE TITLE
For completeness we should describe how to remove the vCenter/VMware data collector

### ISSUE THREAD

**Issue by michaelrstewart1** [NetAppDocs/active-iq-unified-manager#34]
### Page URL

https://docs.netapp.com/us-en/active-iq-unified-manager/storage-mgmt/task_view_and_add_vcenter_servers.html

### Page title

Viewing and adding vCenter Server

### Summary

For completeness we should describe how to remove the vCenter/VMware data collector

For comparison, in the documentation section for adding a cluster data collector we also have [a section](https://docs.netapp.com/us-en/active-iq-unified-manager/storage-mgmt/task_remove_clusters.html) that describes how to remove the cluster from monitoring. We should also have a similar small section [here ](https://docs.netapp.com/us-en/active-iq-unified-manager/storage-mgmt/task_view_and_add_vcenter_servers.html) to inform users of how to remove the vCenter/VMware data source as well. 

A question came in asking about this. The process is intuitive but our documentation is not consistent in the level of detail between cluster data source management and VMware data source management. 

TLDR 
We need to add a section that briefly describes how to remove the VMware/vCenter data sources.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-shwetav** [comment]
Thanks for writing in. We are investigating the issue and will update it with the resolution soon.

**Comment by netapp-shwetav** [comment]
We have added a new topic with the name 'Removing vCenter Server'. It will be available with the upcoming release.

**Comment by netapp-shwetav** [comment]
Hi @michaelrstewart1,

As requested, we have added a new topic for removing vCenter Server.
Link: https://docs.netapp.com/us-en/active-iq-unified-manager/storage-mgmt/task_remove_vcenter_server.html
Thank you for your valuable feedback.

Regards,
Shweta
